docs/other/troubleshooting-gratia.md

```diff
@@ -87,7 +91,7 @@ The options you see here are:
 | SSLHost             | The GRACC server this probe reports to                                                |
 | SSLRegistrationHost | The GRACC server this probe reports to                                                |
 | ProbeName           | The unique name for this probe. Note that it includes the probe type and the host name |
-| SiteName            | The name of your site, as registered in OIM. If your site must be registered in OIM    |
+| SiteName            | The name of your Resource, as registered in [OSG Topology](/common/registration).      |
```

Comment:
```suggestion
| SiteName            | The name of your Resource, as registered in [OSG Topology](../common/registration.md).      |
```

New MkDocs version, new link format requirements: https://opensciencegrid.org/technology/documentation/style-guide/#links

Reply:
>MkDocs 1.0.0 does not support site-relative links (e.g. /software/development-process.md). You must use document-relative links (e.g. ../software/development-process.md) instead.

What a slap in the face, MkDocs.